src/ga4gh/vrs/extras/annotator/vcf.py

```diff
@@ -33,29 +33,32 @@ class SeqRepoProxyType(str, Enum):
     REST = "rest"
 
 
+# Field names for VCF
```

Comment:
Wonder if we should just make VCF Info Enum

Reply:
@korikuzma good point, I'll do that tomorrow